backend/src/apiserver/storage/experiment_store.go

```diff
@@ -188,6 +189,11 @@ func (s *ExperimentStore) DeleteExperiment(id string) error {
 		tx.Rollback()
 		return util.NewInternalServerError(err, "Failed to delete experiment %s from table", id)
 	}
+	err = s.defaultExperimentStore.ClearDefaultExperimentId(tx, id)
```

Comment:
how about naming this wipeoutExperimentIdIfDefault or something equvalent. 
with this name it's a bit hard to follow without context

Reply:
Yeah, you're very right. How about `UnsetDefaultExperimentIdIfIdMatches` ?